### PR TITLE
Styles

### PR DESCRIPTION
**Pull request by anoblet**
https://app.clickup.com/t/86b1dy579

### CODE REVIEW

File: packages/cxl-ui/scss/cxl-course-card.scss [hunk 1]
```diff
@@ -1,10 +1,11 @@
 /* stylelint-disable value-no-vendor-prefix, property-no-vendor-prefix -- some of these are necessary for line-clamp implementation */
-/* stylelint-disable declaration-no-important  */
+/* stylelint-disable declaration-no-important, order/properties-order */
```
Comment (pawelkmpt) [inline packages/cxl-ui/scss/cxl-course-card.scss:2]: why properties-order rule is bad and you removed it?

**Review by pawelkmpt — CHANGES_REQUESTED** [review]
Please also commit "Increase font size" styles which are currently in institute customizer

File: packages/cxl-ui/scss/cxl-course-card.scss (hunk 1 above)
Comment (anoblet) [reply packages/cxl-ui/scss/cxl-course-card.scss:2]: It's too much of a headache remembering the CSS Spec. I prefer alphabetical.

Comment (pawelkmpt) [reply packages/cxl-ui/scss/cxl-course-card.scss:2]: Isn't it corrected automatically by linter?

**Comment by pawelkmpt** [comment]
> Please also commit "Increase font size" styles which are currently in institute customizer

@anoblet follow up

**Comment by anoblet** [comment]
Will get it done by EOD.

File: packages/cxl-ui/scss/cxl-course-card.scss (hunk 1 above)
Comment (anoblet) [reply packages/cxl-ui/scss/cxl-course-card.scss:2]: It's not :/ I don't mind using the default rules, though we could also use this? https://github.com/hudochenkov/stylelint-order/tree/master/rules/properties-alphabetical-order

**Comment by anoblet** [comment]
We have these styles in customizer, which conflict with what's in this PR:

```
details {
  margin: var(--lumo-space-m) 0;
}

details summary {
  font-weight: 900;
  margin-bottom: var(--lumo-space-s);
}
```

I believe that we should remove these from customizer and go with what's in this PR.

**Comment by pawelkmpt** [comment]
> We have these styles in customizer, which conflict with what's in this PR:
> 
> ```
> details {
>   margin: var(--lumo-space-m) 0;
> }
> 
> details summary {
>   font-weight: 900;
>   margin-bottom: var(--lumo-space-s);
> }
> ```
> 
> I believe that we should remove these from customizer and go with what's in this PR.

Where do you see the conflict?
Native `details` are used for FAQ, e.g. https://cxl.com/institute/programs/conversion-optimization/
and what you pushed here is for `vaadin-details` https://cxl.com/institute/lesson/where-and-what-to-test-to-maximize-roi/

**Review by pawelkmpt — APPROVED** [review]

**Pull request merged**